packages/internal/src/storage-test-utils.js

```diff
@@ -74,17 +130,10 @@ harden(makeFakeStorageKit);
 
 export const makeMockChainStorageRoot = () => {
   const { rootNode, data } = makeFakeStorageKit('mockChainStorageRoot');
-
-  const defaultMarshaller = makeMarshal(undefined, (_slotId, iface) => ({
-    iface,
-  }));
-
   return Far('mockChainStorage', {
     ...bindAllMethods(rootNode),
     /**
-     * Defaults to deserializing pass-by-presence objects into { iface } representations.
-     * Note that this is **not** a null transformation; capdata `@qclass` and `index` properties
-     * are dropped and `iface` is _added_ for repeat references.
+     * Defaults to deserializing pass-by-presence objects into Remotable objects.
```

Comment:
We no longer use the term "pass-by-presence objects", instead just calling them "remotable". With this rephrased the comment becomes almost a tautology, and unnecessary. Feel free to delete.

Reply:
hm, I was struggling to find a better word than "object" for the first appearance there. In the earlier format I would have said "defaults to deserializing `@qclass: 'slot'` objects into Remotable objects", because in pre-smallcaps the capdata did indeed contain object records (with the `@qclass` property), hence the first "object" in the sentence. In smallcaps, these records are strings.. maybe I should try "deserializing `$slot` capdata nodes".

The bespoke unserializer that this replaced allowed us to deserialize such things into non-Remotables like `{ iface }` or `{ slot }`. Of course marshal doesn't let us do that, but the non-default `convertSlotToVal` that this takes might produce Remotables with an extra `getBoardID()` method, or the slot string embedded in the iface. So the second part of the sentence should probably say "plain Remotable objects", and mention what a non-plain one might be.

I'll see if I can rephrase it.

Reply:
```
     * Defaults to deserializing slot references into plain Remotable
     * objects, but if supplied with a different marshaller, it could
     * produce Remotables with e.g. the slog string embedded in the
     * iface.
```

Reply:
> the slog [sic] string

Do you mean "the _slot_ string"?

Reply:
gah, yes, apparently my fingers are hardwired to type "slog". PR inbound.